app/views/admin/shelters/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>All Shelters</h2>
+
+<% @shelters.each do |shelter| %>
+  <div id="shelter-<%= shelter.id %>">
```

Comment:
This formatting looks set up well to use within blocks. I wonder if the issue you had arose from a syntax issue. The syntax can be tricky!! I would save this line to remember what the syntax can look like. What trips me up a lot is that the erb tags here have to be render tags (i.e. you need the `=`) even though the div tag is not going to render on the page. The ruby needs to return a value to populate the div id tag.